CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ You may also find the [Upgrade Guide](https://rust-random.github.io/book/update.
 - Rename `rand::thread_rng()` to `rand::rng()`, and remove from the prelude (#1506)
 - Remove `rand::random()` from the prelude (#1506)
 - Rename `Rng::gen_range` to `random_range`, `gen_bool` to `random_bool`, `gen_ratio` to `random_ratio` (#1505)
+- Rename `Uniform` to `StandardUniform` (#1526)
```

Comment:
It's `Standard` to `StandardUniform`

Reply:
It just shows that code reviews are not useless!